example/web/index.html

```diff
@@ -9,29 +24,19 @@
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
   <meta name="apple-mobile-web-app-title" content="example">
-  <link rel="apple-touch-icon" href="/icons/Icon-192.png">
+  <link rel="apple-touch-icon" href="icons/Icon-192.png">
 
   <!-- Favicon -->
-  <link rel="shortcut icon" type="image/png" href="/favicon.png"/>
+  <link rel="icon" type="image/png" href="favicon.png"/>
 
-  <!-- Maplibre -->
-  <script src='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.js'></script>
-  <link href='https://unpkg.com/maplibre-gl@latest/dist/maplibre-gl.css' rel='stylesheet' />
+  <title>MapLibre Flutter Example</title>
+  <link rel="manifest" href="manifest.json">
 
-  <title>example</title>
-  <link rel="manifest" href="/manifest.json">
+  <!-- MapLibre -->
+  <script src='https://unpkg.com/maplibre-gl@^4.3/dist/maplibre-gl.js'></script>
+  <link href='https://unpkg.com/maplibre-gl@^4.3/dist/maplibre-gl.css' rel='stylesheet' />
 </head>
 <body>
-  <!-- This script installs service_worker.js to provide PWA functionality to
-       application. For more information, see:
-       https://developers.google.com/web/fundamentals/primers/service-workers -->
-  <script>
-    if ('serviceWorker' in navigator) {
-      window.addEventListener('load', function () {
-        navigator.serviceWorker.register('/flutter_service_worker.js');
-      });
-    }
-  </script>
-  <script src="main.dart.js" type="application/javascript"></script>
+  <script src="flutter_bootstrap.js" async></script>
```

Comment:
Do you know in which Flutter version this bootstrap script was added.

Reply:
It got added in 3.22.

Reply:
Ok but then we need to bump the Flutter version in the pubspec?

Reply:
Could be, sorry I don't know if there is any backwards compatibility. But because this does only affects the example app the version bump shouldn't be that much of a program, right? However, because we are using an outdated version in the CI, it will probably fail if we increase the min version.